src/otx/config/eval.yaml

```diff
@@ -0,0 +1,18 @@
+# @package _global_
```

Comment:
Not quite sure if we are using this `eval.yaml`, maybe remove it if it's not used.

Reply:
Removed